package.json

```diff
@@ -20,7 +20,6 @@
   "scripts": {
     "jsdoc": "jsdoc src/**/*.js --destination docs --configure .jsdoc --pedantic --readme CONTRIBUTING.md --package package.json",
     "lint": "eslint src/ tasks/ test/ mbTest/ bin/mb && node tasks/lints/licenseCheck.js && node tasks/lints/deadCheck.js && node tasks/lints/objectCheck.js",
-    "postinstall": "cd mbTest && npm install",
```

Comment:
removed postinstall to avoid infinite loop between mountebank and mbTest package